### PR TITLE
Build : cart-page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
eec6a7ca-0b55-4b9b-b744-eb888534e9ea

### CODE REVIEW

**Comment by arjun-computer-geek** [comment]
give enough padding in card

**Review by arjun-computer-geek — APPROVED** [review]
else looks good to me

File: pages/cart-page.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+                <div class="nav-icons">
+                    <a href="/pages/login-page.html">
+                        <i class="fa fa-user"></i>
+                        <span class="nav-icon-text">Login</span>
+                    </a>
+                    <a href="#">
+                        <i class="fa fa-heart" aria-hidden="true"></i>
+                        <span class="nav-icon-text">Wishlist</span>
+                    </a>
+                    <a href="">
```
Comment (itsmebipul00) [inline pages/cart-page.html:44]: href to ?

File: pages/cart-page.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                                    class="fab fa-github-square"></i></a></li>
+                        <li><a href="https://twitter.com/shubhamkr_26" target="_blank" class="social-link"><i
+                                    class="fab fa-twitter-square"></i></a></li>
+                        <li><a href="https://www.instagram.com/i_subhu/" target="_blank" class="social-link"><i
+                                    class="fab fa-instagram"></i></a></li>
+                        <li><a href="https://www.facebook.com/shubham.Lnctu" target="_blank" class="social-link"><i
+                                    class="fab fa-facebook-square"></i></a></li>
+                        <li><a href="https://www.linkedin.com/in/shubham-kumar-6625b8217/" target="_blank"
+                                class="social-link"><i class="fab fa-linkedin"></i></a></li>
+
```
Comment (Megha-Joshi) [inline pages/cart-page.html:186]: Give proper spacing

**Review by itsmebipul00 — APPROVED** [review]
Looks nice, good to merge

File: styles/cart-page.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    justify-content: space-around;
+    align-items: center;
+}
+.cart-price-container {
+    border: 2px solid ;
+    box-shadow: 1px 1px 1px 1px var( --ecomm-wishlist-color);
+    width: 24rem;
+    padding: 2rem;
+    height: fit-content;
+
```
Comment (Megha-Joshi) [inline styles/cart-page.css:22]: give proper spacing

**Review by Megha-Joshi — APPROVED** [review]
Give proper spacing between tags.

File: pages/cart-page.html (hunk 1 above)
Comment (Shubhamkr2610) [reply pages/cart-page.html:44]: Okay ,I'll add to this .

File: pages/cart-page.html (hunk 2 above)
Comment (Shubhamkr2610) [reply pages/cart-page.html:186]: Okay, I'll fix  this . Thank you

File: styles/cart-page.css (hunk 3 above)
Comment (Shubhamkr2610) [reply styles/cart-page.css:22]: Okay, I'll look in it . Thank you

**Pull request merged**